resources/lang/en/model_validation.php

```diff
@@ -24,7 +24,19 @@
     'too_long' => ':attribute exceeded maximum length - can only be up to :limit characters.',
     'wrong_confirmation' => 'Confirmation does not match.',
 
-    'beatmap_discussion' => [
+    'beatmap_discussion_post' => [
+        'first_post' => 'Can not delete starting post.',
+    ],
+
+    'beatmapset_discussion' => [
+        'beatmap_mismatch' => "Difficulty and beatmap don't match.",
```

Comment:
What does this mean? Won't deleted beatmaps trigger this too? 😕

Also, how many of these errors can be triggered by users 'normally'? 'cuz in those cases, they should probably be more helpful instead of "x fails constraint y"?